assets/css/variables.scss

```diff
@@ -0,0 +1,10 @@
+---
+---
+
+$primary: {{- site.colors.primary-color -}};
+$secondary: {{- site.colors.secondary-color -}};
+$secondary-hover: lighten($secondary, 10%);
+
+.has-text-secondary-hover:hover {
+	color: $secondary-hover;
+}
```

Comment:
would this actually work? i thought from experimentation just now that the only way to inject liquid variables into scss was thru the `custom_css_path`?

Reply:
To add on: Is it possible to use the variables within the SCSS partials files? My understanding here is that `styles.scss` and `variables.scss` will be considered separate "projects" and the variables from `variables.scss` cannot be used in `styles.scss`.

Reply:
Manually verified that this works - the same concept is actually used in `misc/custom.scss` which already works - the reason it wasn't working earlier was because we were trying to import it in a different file which wasn't compiled by jekyll, but here we explictly link this as a stylesheet, so it ends up correctly parsed.

I tried to get variables working within the SCSS partials, but couldn't quite get it to work - it seems like it may be possible (see https://github.com/JustGoodThemes/Scriptor-Jekyll-Theme), but for now i think we can stick to `has-text-secondary` and equivalents? I added `has-text-secondary-hover` to help with the other colour that we might commonly use